Polygon_repair_2/examples/Polygon_repair_2/repair_polygon_2.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <filesystem>
+
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Polygon_repair_2/Polygon_repair_2.h>
+#include <CGAL/draw_polygon_2.h>
+#include <CGAL/draw_polygon_with_holes_2.h>
+#include <CGAL/Polygon_repair_2/draw_multipolygon_with_holes_2.h>
+#include <CGAL/IO/WKT.h>
+
+using Kernel = CGAL::Exact_predicates_inexact_constructions_kernel;
+using Point_2 = Kernel::Point_2;
+using Polygon_2 = CGAL::Polygon_2<Kernel>;
+using Polygon_with_holes_2 = CGAL::Polygon_with_holes_2<Kernel>;
+using Multipolygon_with_holes_2 = CGAL::Multipolygon_with_holes_2<Kernel>;
+using Polygon_repair_2 = CGAL::Polygon_repair_2::Polygon_repair_2<Kernel>;
+
+int main(int argc, char* argv[]) {
+
+  for (const auto& file: std::__fs::filesystem::directory_iterator("../../test/Polygon_repair_2/data/in")) {
+    std::cout << "Reading " << file.path().filename() << "..." << std::endl;
+
+    // if (file.path().filename() != "hole-outside.wkt") continue;
+
+    std::string in;
+    std::getline(std::ifstream(file.path()), in);
+    std::istringstream iss(in);
+    std::size_t is_polygon = in.find("POLYGON");
+    std::size_t is_multipolygon = in.find("MULTIPOLYGON");
```

Comment:
Is it intentional that when it is `MULTIPOLYGON` als `is_polygon` will be `true`?

Reply:
Ah, it is not a  `bool` but the position.  I was mislead by the `is_` prefix. Maybe swith to `bool`  and do the `==0`  right away.